android/src/main/java/com/brentvatne/exoplayer/ExoPlayerView.java

```diff
@@ -281,17 +219,17 @@ public void onVideoSizeChanged(VideoSize videoSize) {
             // React native workaround for measuring and layout on initial load.
             if (isInitialRatio) {
                 post(measureAndLayout);
-            }
+            }*/
         }
 
         @Override
         public void onRenderedFirstFrame() {
-            shutterView.setVisibility(INVISIBLE);
+            setHideShutterView(hideShutterView);
         }
 
         @Override
         public void onTracksChanged(@NonNull Tracks tracks) {
-            updateForCurrentTrackSelections(tracks);
+            //updateForCurrentTrackSelections(tracks);
```

Comment:
In fact this call was here to avoid the blinking I reported.
The goal was to resize the view as soon as possible when we receive new aspect ratio